neural_compressor/common/utility.py

```diff
@@ -25,3 +25,4 @@
 # config name
 BASE_CONFIG = "base_config"
 RTN_WEIGHT_ONLY_QUANT = "rtn_weight_only_quant"
+KERAS_STATIC_QUANT = "keras_static_quant"
```

Comment:
Since the `neural-compressor[tf]` binary only includes tf-related things, it's unnecessary to use `KERAS` prefix to distinguish frameworks such as `torch` and `keras`.